src/CentreonModule/Application/Webservice/CentreonModuleWebservice.php

```diff
@@ -51,6 +51,19 @@ class CentreonModuleWebservice extends Webservice\WebServiceAbstract implements
     Webservice\WebserviceAutorizeRestApiInterface
 {
 
+    /**
+     * Check ACL rights on extension/manager page if user is not admin
+     */
+    private function checkAcl(): void
```

Comment:
there is already a `authorize` method which allows to check acl.
You should averride this method which is defined in `CentreonWebService`